### PR TITLE
Add lr scheduler callback for high level api

### PR DESCRIPTION
**Pull request by LielinJiang**
### PR types

New features
### PR changes

APIs
### Describe

Add lr scheduler callback for high level api

### CODE REVIEW

File: python/paddle/hapi/callbacks.py [hunk 1]
```diff
@@ -485,6 +491,95 @@ def on_train_end(self, logs=None):
...
+            train_dataset = paddle.vision.datasets.MNIST(mode='train', transform=transform)
+
+            lenet = paddle.vision.LeNet()
+            model = paddle.Model(lenet,
+                inputs, labels)
+
+            base_lr = 1e-3
+            boundaries = [5, 8]
+            wamup_steps = 4
+            def make_optimizer(parameters=None):
```
Comment (saxon-zh) [inline python/paddle/hapi/callbacks.py:526]: 这个函数没有调用的地方

Comment (LielinJiang) [reply python/paddle/hapi/callbacks.py:526]: done, thanks

**Review by qingqing01 — APPROVED** [review]
LGTM.

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by saxon-zh — APPROVED** [review]
LGTM
